### ISSUE TITLE
The ctrl-backspace command doesn't read the deleted word in LibreOffice Writer

### ISSUE THREAD

**Issue by bingen13** [nvaccess/nvda#15436]
### Steps to reproduce:

Open LibreOffice Writer. I have tested with version 7.6.0.6 64 bits, but I don't think it especially matters:

Write a few words, for example: NVDA is a free screen reader.

Then press control-backspace a couple of times.

### Actual behavior:

````
Untitled 1 — LibreOffice Writer
Untitled 1 - LibreOffice Document  document  editable
blank
speak command keys on
NVDA
is
a
free
screen
reader
ctrl+backspace
dot
ctrl+backspace
reader
ctrl+backspace
space
ctrl+backspace
space
ctrl+backspace
space
ctrl+backspace
space
ctrl+backspace
space
ctrl+backspace

````

The dot and the word reader were correctly announced, but not the others.

### Expected behavior:

````
ctrl+backspace
dot
ctrl+backspace
reader
ctrl+backspace
screen
ctrl+backspace
free
ctrl+backspace
a
ctrl+backspace
is
ctrl+backspace
sNVDA
ctrl+backspace

````

Each press of ctrl-backspace should announce the word that is being deleted.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2023.2

#### Windows version:
Windows 10 22H2 (AMD64) build 19045.3324

#### Name and version of other software in use when reproducing the issue:

````
Version: 7.6.0.3 (X86_64) / LibreOffice Community
Build ID: 69edd8b8ebc41d00b4de3915dc82f8f0fc3b6265
CPU threads: 8; OS: Windows 10.0 Build 19045; UI render: Skia/Raster; VCL: win
Locale: en-GB (es_ES); UI: en-GB
Calc: CL threaded
````

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have not.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by Qchristensen** [comment]
I can also replicate this using:

NVDA 2023.2 and also NVDA 2021.3.5
In Libre Office 7.5.1.2 (X86_64)
on Windows 11 (64-bit) Version: 22H2, Build: 22621.2134

**Comment by XLTechie** [comment]
CC @michaelweghorn

**Comment by michaelweghorn** [comment]
Potential fix: #15558
